changes/7194.added

```diff
@@ -1 +1 @@
-Added `JobRunScheduleButton` class to `nautobot.extras.views`.
+Added `jobs_ui.py` to `nautobot.extras.views` with classes: `JobRunScheduleButton`, `JobKeyValueOverrideValueTablePanel` and `JobObjectFieldsPanel` and `render_job_queues_list` helpers method.
```

Comment:
This isn't a user-facing change, so I'd suggest either removing it or moving to `.housekeeping` if you feel it's important to document.